tests/utils/env/test_env_manager.py

```diff
@@ -1068,7 +1068,7 @@ def test_create_venv_uses_patch_version_to_detect_compatibility(
 
     m.assert_called_with(
         config_virtualenvs_path / f"{venv_name}-py{version.major}.{version.minor}",
-        executable=None,
+        executable=Path(sys.executable),
```

Comment:
Why did that change? Did you change something in the algorithm?

Reply:
Previously the `create_venv` assumed, that unless an `executable` was given or we detected the active python, the "default" python was used. Now we use an instance of `Python` in any way. See https://github.com/python-poetry/poetry/pull/9050/files#diff-37f4bd765a025d97c1128f60e349dc94b7df596ac9c9dd8293b7b4ad4e6bafb3R405